spiketoolkit/postprocessing/postprocessing_tools.py

```diff
@@ -1494,29 +1495,7 @@ def _select_max_channels_from_waveforms(wf, recording, max_channels):
         max_channel_idxs = np.arange(recording.get_num_channels())
 
     return max_channel_idxs
-
-
-def _select_max_channels_from_templates(template, recording, max_channels):
```

Comment:
can you leave this? I might need it for other functions :)

Reply:
ok